releasenotes/notes/csm-7-50-14413f70a64656f9.yaml

```diff
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    CWS: Ability to send a signal to a process when a rule was triggered.
+    CWS: Add Kubernetes user session context to events, in particular the username, UID and groups of the user that ran the commands remotely.
+
+enhancements:
+  - |
+    CWS: Support for Ubuntu 23.10.
+    CWS: Support using TCP in addition in UNIX sockets.
+    CWS: Reduce memory usage of ring buffer on machines with large memory capacities.
```

Comment:
```suggestion
    CWS: Reduce memory usage of ring buffer on machines with more than 64 CPU cores.
```

Reply:
Good point. I think it's 16 CPU cores though

Reply:
The memory usage for the ]16;64] bucket didn't change, it's still `16 * 256 * os.Getpagesize()` https://github.com/DataDog/datadog-agent/pull/19940/files

Reply:
Yes it's only 64 and more that changed

Reply:
Oh right, I read too fast